scripts/zkai_d128_rmsnorm_to_projection_bridge_input.py

```
@@ -0,0 +1,527 @@
+    except (OSError, UnicodeDecodeError, json.JSONDecodeError) as err:
+        raise D128BridgeInputError(f"failed to load {label}: {err}") from err
+    if not isinstance(payload, dict):
+        raise D128BridgeInputError(f"{label} must be an object")
+    return payload
+
+
+def load_target(path: pathlib.Path = TARGET_JSON) -> dict[str, Any]:
+    target = load_json(path, MAX_TARGET_JSON_BYTES, "d128 target evidence")
+    summary = target.get("summary")
+    if not isinstance(summary, dict):
+        raise D128BridgeInputError("target summary missing")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)